tests/e2e/utils/androidReversePort.ts

```diff
@@ -1,6 +1,8 @@
 import config from '../config';
 import execAsync from './execAsync';
 
-export default function () {
+function androidReversePort(): Promise<void> {
```

Comment:
In my Draft PR with execAsync migration I've created [a return type](https://github.com/Expensify/App/pull/36922/files#diff-0c1ace095ae524ef2a0a0ec3aa72dd762c577115c1c358dfb4446bddc5e2e997R5) of the execAsync function. Maybe this type will be more correct to return, but it's still in the internal review

Reply:
Let's hold this PR until yours is merged 👍

Reply:
@blazejkustra The PR with execAsync migration is merged and you can reuse the return type now